bindings/cs/rl.net.cli/EntryPoints.cs

```diff
@@ -9,8 +9,9 @@ static class EntryPoints
         public static void Main(string [] args)
         {
             //BasicUsageExample(args);
+            PdfPassthroughExample(args);
```

Comment:
>PdfPassthroughExample [](start = 12, length = 21)

This is a bit out of scope for this PR but instead of modifying the code to run the examples can we just add an `args` value to trigger the appropriate example?

Reply:
Instead of adding another arg maybe the it can be offered as an option to select once the program starts ?

Reply:
Added #48 and #49 to track this (and related work) to clean up the usage/purpose of RL.Net.Cli